tests/commoncode/test_filetype.py

```diff
@@ -76,6 +76,11 @@ def test_get_type(self):
 
         assert sorted(expected) == sorted(results)
 
+    def test_get_debiantype(self):
+        test_file = self.get_test_loc('filetype/debian/libjama-dev_1.2.4-2_all.deb')
+        type = filetype.get_type(test_file)
+        assert 'f' == type 
+
```

Comment:
@linexb I think this test is not needed. It just test that a .deb package is not a special file so this is not super interesting. I think you can remove it and the corresponding test file